### ISSUE TITLE
Compile JSX for distribution

### ISSUE THREAD

**Issue by williamboman** [vadimdemedes/react-floating-label-input#1]
Currently I have to explicitly tell webpack to use the babel-loader (with my babel and JSX configurations) in order to use this component;

```js
import ReactFloatingLabel from 'babel-loader!react-floating-label-input'
```

**Comment by vadimdemedes** [comment]
Yes, it should've been done initially. I wasn't very experienced with React back then when I created this library. Would you be willing to set up babel in a PR?
